### PR TITLE
Changed confusing vm controllerAs name

### PR DESCRIPTION
**Pull request by termosfera**
I found myself working in a huge project with the whole view full of vm.whatever. When I ask my team mates why did they do that, they told me "It´s in John´s Papa´s styleguide" because of this misconception at this point of the guide. If I´m wrong sorry and let me know. Thanks.

### CODE REVIEW

**Comment by mikeerickson** [comment]
I define the name of my controllerAs to be most appropriate for actually controller actions (for example customerController is 'cust') 

The point of "vm" is merely an idea or indication that it is a "view model" but not the rule for naming the controller

In your example, this is a clear reason as to why NOT to use "vm" everywhere as you be able to identify what controller you are referencing 

TL;DR Use logical controller names when defining "controllerAs" :)

**Comment by termosfera** [comment]
Yes, you are right, and I can see it clear, but seems confusing for some people. The guide is in an imperative style (as every style guide should be) and that vm looks like the way it has to be.

**Comment by mtpultz** [comment]
Hi @termosfera, it states this in the guide:

> Note: When working with larger codebases, using a more descriptive name can help ease cognitive overhead & searchability. Avoid overly verbose names that are cumbersome to type.

https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y032
